app/components/explore/dynamic-explore/Hero.tsx

```diff
@@ -0,0 +1,31 @@
+import Image from "next/image";
+
+const Hero = ({ level }: { level: string }) => {
+  const imageUrl = `/assets/images/explore/level/${level}.png`;
+
+  return (
+    <section className="relative">
+      <div
+        className={`bg-[url("/assets/images/explore/level/hero_background.png")] relative rounded-tl-2xl overflow-hidden bg-cover bg-center `}
+      >
```

Comment:
classname doesn't need to be wrapped in curly braces, it can be a string, also remove whitespace